separate defaults.json and configurations for semeru

mahdipub opened the pull request:
By moving configurations to Semeru_configuration and also create a semeru_default.json tailored for semeru, it help reduce merge conflicts from upstream and have a more smooth merge. If merged it will reset all configurations and defaults.json to current from adoptium and moves current one for semeru to separate file/folders that will not be hit by adopt changes. related automation#265.

Signed-off-by: mahdi@ibm.com

mahdipub committed:
separate defaults.json and configurations for semeru

By moving configurations to Semeru_configuration and also create a semeru_default.json tailored for semeru, it help reduce merge conflicts from upstream and have a more smooth merge. If merged it will reset all configurations and defaults.json to current from adoptium and moves current one for semeru to separate file/folders that will not be hit by adopt changes. related automation#265.

Signed-off-by: mahdi@ibm.com

AdamBrousseau reviewed (commented):
I'm not sure if we need the evaluation and release files for Semeru since we don't use them. We can add them later if we need them.

Also I would add a second commit to cleanup all the temurin parts from the semeru configs.

mahdipub committed:
remove temurin from semeru configurations

removes temurin entries from semeru separated configurations files under `Semeru_configuration` as those are not used by Semeru. Related automation_265

Signed-off-by: mahdi@ibm.com

mahdipub commented:
That is done. 
2nd commit added and temurin entries removed.
